packages/types/src/type/Vote.spec.ts

```diff
@@ -17,15 +19,76 @@ describe('Vote', () => {
     expect(new Vote(new Boolean(true)).isAye).toBe(true);
   });
 
-  it('has isYay for negative', () => {
+  it('has isNay for negative', () => {
     expect(new Vote(new Boolean(false)).isNay).toBe(true);
   });
 
-  it('is positive for negative numbers', () => {
+  it('is Aye for negative numbers', () => {
     expect(new Vote(-999).isAye).toBe(true);
   });
 
-  it('is negative for positive numbers', () => {
+  it('is Nay for positive numbers', () => {
     expect(new Vote(999).isNay).toBe(true);
   });
+
+  it('constructs V2 Vote with raw boolean', () => {
+    expect(
+      new Vote({
+        aye: true,
+        conviction: new Conviction('Locked1x')
+      })
+      .toU8a()
+    )
+      .toEqual(new Uint8Array([1, 0, 0, 0, 0, 0, 0, 1]));
+  });
+
+  it('constructs with Vote aye is false, conviction is None', () => {
+    expect(
+      new Vote({
+        aye: new Boolean(false),
+        conviction: new Conviction('None')
+      })
+      .toU8a()
+    )
+      .toEqual(new Uint8Array([0, 0, 0, 0, 0, 0, 0, 0]));
+  });
+
+  it('constructs with Vote aye is true, conviction is Locked4x', () => {
+    expect(
+      new Vote({
+        aye: new Boolean(true),
+        conviction: new Conviction('Locked4x')
+      })
+      .toU8a()
+    )
+      .toEqual(new Uint8Array([1, 0, 0, 0, 0, 1, 0, 0]));
```

Comment:
this is incorrect!

it should be `new Uint8Array([0b10000100])`, i.e. vote.toU8a() is *always* one byte